starport/pkg/localfs/search_test.go

```diff
@@ -0,0 +1,124 @@
+package localfs
+
+import (
+	"io/fs"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"syscall"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func setupGlobTests() (string, error) {
+	dir, err := os.Getwd()
+	if err != nil {
+		return "", err
+	}
+	tmpdir, err := ioutil.TempDir(dir, "glob-test")
+	if err != nil {
+		return "", err
+	}
+	dirs := []string{"foo/bar", "foo/baz"}
+	for _, dir := range dirs {
+		err = os.MkdirAll(filepath.Join(tmpdir, dir), 0755)
+		if err != nil {
+			return "", err
+		}
+	}
+	files := []string{
+		"foo/file.proto",
+		"foo/bar/file1.proto",
+		"foo/bar/file2.proto",
+		"foo/baz/file.proto",
+		"foo/file",
+		"foo/baz/file",
+	}
+	for _, file := range files {
```

Comment:
Instead of MkdirAll'ing above, we can `dir := filepath.Dir(file)` and `MkdirAll(dir)` to make things a bit more flexible.